### ISSUE TITLE
[a11y Support] Accredited Representation Management: Widget - Pre-Staging Review

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#78465]
This ticket is for the QA of the Representative Status Widget, prior to the ARM team heading into Staging Review.

Launch will include placement on:
1. [The accredited rep landing page](https://www.va.gov/get-help-from-accredited-representative/)
2. [The Find a Rep product page](https://www.va.gov/get-help-from-accredited-representative/find-rep/)

And the team is aiming to have test users for each of the [Widget states](https://www.figma.com/file/bzbwObT9hiItve0q3cQX9c/Find-and-Appoint-a-Representative?type=design&node-id=1283-176978&mode=design&t=dTXkE68yvO7n29fF-0)

QA will likely be running through the Find a Rep product page with test users to ensure the Widget content and placement matches expectation.  Any accessibility findings can be shared with the team!

Currently unsure if there is a way to test the landing page placement, as that page is owned by CAIA.

### Timing
According to the current [roadmap](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1695926957756/3bcc5202bdc4d93a6898b5bc114b3a45461ab3f7?sender=ubac5f0487f25bc4431288699):
- Internal QA on 4/9, staged rollout 4/30-5/13
- Ideally CAIA would deliver a spreadsheet with findings by EOD April 12

### Definition of done

- [x] Receives link to staging environment
- [x] Receives test user account information
- [x] Creates an [audit document](https://docs.google.com/spreadsheets/d/1RuanAgzRvsjcugODcKvxWBLyUsYq7wiVfy7XeogNX1E/edit#gid=1370873395) encompassing all widget states and placement
- [x] Audits
- [x] Shares findings with ARM team

**Comment by oddball-lindsay** [comment]
@coforma-jamie we are ready for QA of our [Rep Status Widget in Staging](https://staging.va.gov/get-help-from-accredited-representative/find-rep/)!  We are taking notes in this [Google Doc](https://docs.google.com/document/d/1IN15xxNxbaEjSTEWLCWRyVbxqtU6XzXjJ4Fw7NQuy7c/edit#heading=h.lwtkdnboora9) and have saved a section for you, but if you prefer to use your GSheet like our last QA round simply drop a link into that GDoc or even here as a comment 🙏 

We have [test users](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/staging-test-accounts-accredited-representation-management-poa.csv) and I have some notes about how to use those test users in [this comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/78093#issuecomment-2061646491).  Currently we do not have a way to QA the "Individual" rep state.

**Comment by coforma-jamie** [comment]
We've completed as much of an audit as our schedules and tech will allow :) Findings have been shared with the team via the [audit spreadsheet](https://docs.google.com/spreadsheets/d/1RuanAgzRvsjcugODcKvxWBLyUsYq7wiVfy7XeogNX1E/edit#gid=1370873395), [QA doc](https://docs.google.com/document/d/1IN15xxNxbaEjSTEWLCWRyVbxqtU6XzXjJ4Fw7NQuy7c/edit), and Slack.

Thanks for working with us @oddball-lindsay! :)

**Comment by oddball-lindsay** [comment]
@coforma-jamie we're still trying to get a test user for the Individual rep state -- hoping to get that by Friday at the latest.

Do you want to keep this ticket open to test that state as well?

**Comment by coforma-jamie** [comment]
Ahh, didn't realize! I'll reopen :) Make sure to let @eli-oat know so he can take a look, since I'll be out until Tuesday.

**Comment by coforma-jamie** [comment]
@oddball-lindsay, we've finished our review! Please check rows 7-9 on the Issues tab of our [spreadsheet](https://docs.google.com/spreadsheets/d/1RuanAgzRvsjcugODcKvxWBLyUsYq7wiVfy7XeogNX1E/edit#gid=0) for our findings.
